dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -36,47 +36,79 @@ def test_train_method_exists(self):
         self.assertTrue(hasattr(dp, 'train_structured_labeler'))
 
         with mock.patch('dataprofiler.train_structured_labeler') as mock_obj:
-            dp.train_structured_labeler(None, None)
-            self.assertEqual(mock_obj.call_args, mock.call(None, None))
+            dp.train_structured_labeler(None, None, None)
+            self.assertEqual(mock_obj.call_args, mock.call(None, None, None))
 
     def test_accepted_inputs(self):
         with self.assertRaisesRegex(TypeError,
                                     "Input data must be either a "
                                     "`pd.DataFrame` or a `data_profiler.Data` "
                                     "and not of type `TextData`."):
-            dp.train_structured_labeler(None, None)
+            dp.train_structured_labeler(None, None, None)
 
         with self.assertRaisesRegex(TypeError,
                                     "The output dirpath must be a string."):
-            dp.train_structured_labeler(pd.DataFrame([]), save_dirpath=0)
+            dp.train_structured_labeler(
+                pd.DataFrame([]), default_label=None, save_dirpath=0)
+
+        with self.assertRaisesRegex(TypeError,
+                                    "Default label must be a string."):
+            dp.train_structured_labeler(
+                pd.DataFrame([]), default_label=1, save_dirpath=None)
 
         # doesn't accept text data
         text_data = dp.Data(data='test', data_type='text')
         with self.assertRaisesRegex(TypeError,
                                     "Input data must be either a "
                                     "`pd.DataFrame` or a `data_profiler.Data` "
                                     "and not of type `TextData`."):
-            dp.train_structured_labeler(text_data, None)
+            dp.train_structured_labeler(
+                text_data, default_label=None, save_dirpath=None)
 
         with self.assertRaisesRegex(ValueError,
                                     "The `save_dirpath` is not valid or not "
                                     "accessible."):
-            dp.train_structured_labeler(pd.DataFrame([]), "/a/test")
+            dp.train_structured_labeler(
+                pd.DataFrame([]), default_label=None, save_dirpath="/a/test")
 
         try:
             data = {'UNKNOWN': ["Beep", "Boop"],
                     'PERSON': ["GRANT", "MENSHENG"]}
             df = pd.DataFrame(data=data)
-            dp.train_structured_labeler(df, save_dirpath=None)
```

Comment:
on this line, since we aren't using `save_dirpath`, lets remove it

Reply:
changed